parsl/tests/test_bash_apps/test_stdout.py

```diff
@@ -34,42 +34,74 @@ def echo_to_streams(msg, stderr=None, stdout=None):
     'bad_mode'
 ]
 
-
-@pytest.mark.issue363
-@pytest.mark.parametrize('spec', speclist, ids=testids)
-def test_bad_stdout_specs(spec):
-    """Testing bad stdout spec cases"""
-
-    fn = echo_to_streams("Hello world", stdout=spec, stderr='t.err')
-
-    try:
-        fn.result()
-    except Exception as e:
-        # This tests for TypeCheckError by string matching on the type name
-        # because that class does not exist in typeguard 2.x - it is new in
-        # typeguard 4.x. When typeguard 2.x support is dropped, this test can
-        # become an isinstance check.
-        assert "TypeCheckError" in str(type(e)) or isinstance(e, TypeError) or isinstance(e, perror.BadStdStreamFile), "Exception is wrong type"
-    else:
-        assert False, "Did not raise expected exception"
-
-
-@pytest.mark.issue363
-def test_bad_stderr_file():
-    """Testing bad stderr file"""
-
-    err = "/bad/dir/t2.err"
-
-    fn = echo_to_streams("Hello world", stderr=err)
-
-    try:
-        fn.result()
-    except perror.BadStdStreamFile:
-        pass
-    else:
-        assert False, "Did not raise expected exception BadStdStreamFile"
-
-    return
+# Skipping these tests temporarily due to root user permissions
```

Comment:
I'll have to do something about this - to merge this PR, this test will have to be re-enabled, so I will try to work on that to unblock things.

Reply:
@mercybassey ok, I think the right thing to do for now here is instead of invoking `make local_thread_test` (which then invokes `pytest parsl/tests/ -k "not cleannet" --config parsl/tests/configs/local_threads.py --random-order --durations 10`), the Github Actions yaml could invoke this command instead:

 `pytest parsl/tests/ -k "not cleannet and not issue363" --config parsl/tests/configs/local_threads.py --random-order --durations 10`

That would skip a bunch of tests related to handling of stderr and stdout - which is fine, as this line is only to verify that basic Parsl runs before running the real tests against flux. For example, taskvine and work queue tests also skip those tests marked `issue363` as well, in our main test run.

Then you don't need to modify this file test_stdout.py at all and can set it back to how it is in `master`